cpp/src/cider/exec/nextgen/operators/RowToColumnNode.cpp

```diff
@@ -127,6 +132,171 @@ class ColumnWriter {
         setNullBuffer(values.getNull(), for_null), raw_length_buffer, raw_data_buffer);
   }
 
+  int32_t getBytesFromSQLType(SQLTypes type) {
```

Comment:
Is there any similar function already exists? If not, where should i move it to?

Reply:
check `inline int64_t getTypeBytes(SQLTypes type) {`?